### PR TITLE
Add sitemap to robots.txt

### PR DESCRIPTION
**Pull request by rap1ds**

### CODE REVIEW

File: lib/rack_middleware/robots_generator.rb [hunk 1]
```diff
@@ -5,13 +5,16 @@ class RobotsGenerator
   # Use the config/robots.txt in production.
   # Disallow everything for all other environments.
   def self.call(env)
+    return [404, {}, []] if env[:current_marketplace].nil?
```
Comment (ovan) [inline lib/rack_middleware/robots_generator.rb]: Why not render the no_index_content in this case?

File: lib/rack_middleware/robots_generator.rb [hunk 2]
```diff
@@ -5,13 +5,16 @@ class RobotsGenerator
   # Use the config/robots.txt in production.
```
Comment (ovan) [inline lib/rack_middleware/robots_generator.rb]: This comment is no longer up to date.

File: lib/rack_middleware/robots_generator.rb (hunk 1 above)
Comment (rap1ds) [inline lib/rack_middleware/robots_generator.rb]: Isn't 404 more accurate information? "This page does not exists" vs "This page does exist but I don't want you to index it"

Comment (ovan) [inline lib/rack_middleware/robots_generator.rb]: Yes, I think it makes sense. The env[:current_marketplace] shouldn't be nil in any sensible scenario.

**Pull request merged**